src/DataModel/Configuration/ItemCrafting/MixResult.cs

```diff
@@ -20,12 +20,12 @@ public enum MixResult
     StaysAsIs = 1,
 
     /// <summary>
-    /// The item will be downgraded to level 0.
+    /// The item will be downgraded to a random level, may lose its skill, and its item option may be reduced by 1 level.
     /// </summary>
-    DowngradedTo0 = 3,
+    ChaosWeaponAndFirstWingsDowngradedRandom = 2,
 
     /// <summary>
-    /// The item will be downgraded to a random level.
+    /// The item will be downgraded 2 or 3 levels and its item option will be removed.
     /// </summary>
-    DowngradedRandom = 4,
+    ThirdWingsDowngradedRandom = 3,
```

Comment:
https://github.com/kyleruss/emu-server/blob/202856a74c905c203b9b2795fd161f564ca8b257/GameServer/Source/ChaosBox.cpp#L4812

Reply:
Also yes. For the same reason as above.